app/src/main/java/com/kidseat/kidseat/EventDetailActivity.java

```diff
@@ -67,10 +73,37 @@ public void onComplete(@NonNull Task<DocumentSnapshot> task) {
                             tvName.setText(documentSnapshot.getString("name"));
                             setTitle(documentSnapshot.getString("name"));
                             tvDate.setText(documentSnapshot.getString("date"));
-                            tvTime.setText(documentSnapshot.getString("time"));
+                            String time = " | " + documentSnapshot.getString("time");
+                            tvTime.setText(time);
                             tvAddress.setText(documentSnapshot.getString("address"));
                             tvDescription.setText(documentSnapshot.getString("description"));
                             tvMealType.setText((documentSnapshot.getString("meal_type")));
+
+                            String facebook_link = documentSnapshot.getString("facebook_link");
```

Comment:
It would be great if we use camel case variable names everywhere for consistency